draft-ietf-httpbis-digest-headers.md

```diff
@@ -581,11 +581,9 @@ digest validation failure at the recipient, preventing the application from
 accessing the representation. Such an attack consumes the resources of both
 endpoints. See also {{digest-and-content-location}}.
 
-Digest fields SHOULD always be used over a connection that provides integrity at
-the transport layer that protects HTTP fields.
-
-A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
-signatures.
+Since signatures are meant to protect against adversarial use cases,
```

Comment:
@jricher PTAL. We already have normative language [here](https://www.ietf.org/id/draft-ietf-httpbis-digest-headers-07.html#section-6-8)